### PR TITLE
Add Stress Buster

### PR DESCRIPTION
**Pull request by supr1yo**

### CODE REVIEW

**Comment by nik132-eng** [comment]
Please do PR after assigning a specific issue, This was not assigned to anyone so this time let it go.

And add the video link on how the extension will work after adding it to the developer mode.

File: Stress Buster/roadmap.md
Comment (nik132-eng) [inline Stress Buster/roadmap.md:1]: Do not delete the roadmap so if anyone wants to update it they can get an idea.

File: Stress Buster/assets/icon.png
Comment (nik132-eng) [inline Stress Buster/assets/icon.png:1]: Add more different sizes of icons of the same image

**Comment by supr1yo** [comment]
The aforesaid changes are done.

**Video demonstration:**

https://github.com/nik132-eng/chrome-extensions/assets/45136370/171ec262-9def-484f-bbb3-56788b2c25fc

**Review by nik132-eng — APPROVED** [review]

**Comment by nik132-eng** [comment]
issue No: #7

**Pull request merged**